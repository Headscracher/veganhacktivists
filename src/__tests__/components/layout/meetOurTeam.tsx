import React from 'react';
import { render } from '@testing-library/react';

import MeetOurTeam from '../../../components/layout/meetOurTeam';

it('should render correctly', () => {
  const { asFragment } = render(<MeetOurTeam />);
  expect(asFragment()).toMatchSnapshot();
});
