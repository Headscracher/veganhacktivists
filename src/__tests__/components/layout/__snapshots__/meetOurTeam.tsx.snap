// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  <div
    class="bg-black"
  >
    <div
      class="content-center mx-auto md:w-1/2 drop-shadow-2xl text-2xl pb-12"
    >
      <p
        class="mb-12 text-grey-dark pt-16"
      >
        <span
          class="text-5xl font-mono uppercase text-white"
        >
          Meet our team
        </span>
      </p>
      <div
        style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
      >
        <div
          style="box-sizing: border-box; display: block; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
            style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
          />
        </div>
        <noscript />
        <img
          alt="Compassion, Creativity, Code"
          data-nimg="true"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
      </div>
      <div
        class="flex justify-center flex-wrap"
      >
        <a
          href="/people/team"
        >
          <div
            class="p-3 px-4 py-2 text-2xl border-l-8 bg-w-x2 ease-out duration-1000 cursor-pointer hover:shadow-fill-green text-grey-dark border-green bg-w-x2 bg-white m-5 font-mono text-sm"
          >
            Learn more
          </div>
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
